ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1360,6 +1360,11 @@ periodics:
       - --bw-allow-unused=ci.dev.openshift.redhat.com
       - --bw-allow-unused=ci.openshift.redhat.com
       - --bw-allow-unused=jira-credentials-dptp-bot
+      - --bw-allow-unused=openshift-bugzilla-robot@redhat.com
+      - --bw-allow-unused=openshift-bot
+      - --bw-allow-unused=openshift-github-robots@redhat.com
+      - --bw-allow-unused=aos-cicd@redhat.com
+      - --bw-allow-unused="aos-team-dp-testplatform@redhat.com mailing list"
```

Comment:
Remove the quotes, this is not going through a shell and the item name does not have quotes in it
/hold

Reply:
thanks, updated. Are we safe with the spaces here?

Reply:
yeah should work as it doesn't go through a shell
/hold cancel